### PR TITLE
Enhancing the guide's utility

### PR DESCRIPTION
**Pull request by lehmannro**
As per my email, I have some changes to the coaching guidelines.  Among some style fixes, I mostly added the keyboard no-no and a note about feedback.

### CODE REVIEW

File: presentations/coaching/index.html [hunk 1]
```diff
@@ -68,19 +68,40 @@
...
         <header>
-          <h2>Coaching isn't teaching...</h2>
+          <h2>A note about the material</h2>
         </header>
-        <p>in the sense that we are not standing in front and teaching here. Coaches</p>
         <p>
           <ul>
+            <li>At times, the material might seem unnecessarily dense to you, as if going through extra hoops.</li>
+            <li>Don't fret, that is on purpose!</li>
+            <li>We believe in teaching <em>the hard way</em>.</li>
```
Comment (gnunicorn) [inline presentations/coaching/index.html]: Please avoid the usage of the word "teaching" as it comes with a long list of not so positive behaviors. How about using "We believe in learning <em>by exploration</em>" - I find that more clear ...

File: presentations/coaching/index.html [hunk 2]
```diff
@@ -68,19 +68,40 @@
...
+            <li>That means teaching them a little unidiomatic style first, just to make them discover <strong>themselves</strong> how (and why!) to do it better.  Example:
```
Comment (gnunicorn) [inline presentations/coaching/index.html]: avoid slang :P . How about "That means letting them use not-so-good style first, just to make them discover..."

File: presentations/coaching/index.html [hunk 3]
```diff
@@ -114,16 +136,17 @@
...
       <section>
         <header>
-          <h2>Creating a friendly environment, part 2</h2>
+          <h2>Creating a friendly environment</h2>
+          Encouragement
         </header>
         <p>
           <ul>
             <li>Assume everyone you're coaching has zero knowledge but infinite intelligence</li>
-            <li>Encourage learners to ask questions</li>
```
Comment (gnunicorn) [inline presentations/coaching/index.html:122]: Why did you remove that line?

File: presentations/coaching/index.html [hunk 4]
```diff
@@ -234,6 +279,7 @@
         </header>
         <p>
           <ul>
+            <li>We do not touch their keyboard</li>
```
Comment (gnunicorn) [inline presentations/coaching/index.html]: I disagree with this one. I don't see why not, if the learner is comfortable with it. This sounds like a very harsh rule especially in case you need to look something up to help them but don't have your own computer.

Comment (gnunicorn) [inline presentations/coaching/index.html]: Ok. with the next paragraph it becomes a bit clearer, but then especially I do not see, why this should be in this list - and on first position for that matter...

Comment (lehmannro) [inline presentations/coaching/index.html]: You're right — it might be misplaced at the very top of the list, when there's much more important advice to be found.  I moved it to the bottom.

File: presentations/coaching/index.html (hunk 3 above)
Comment (lehmannro) [inline presentations/coaching/index.html:122]: It did not fit on the slide (the progress bar covers it in my browser) and the next few slides are all dedicated to only that.

File: presentations/coaching/index.html (hunk 2 above)
Comment (lehmannro) [inline presentations/coaching/index.html]: Done!

File: presentations/coaching/index.html (hunk 1 above)
Comment (lehmannro) [inline presentations/coaching/index.html]: Valid point;  fixed.

File: presentations/coaching/index.html (hunk 3 above)
Comment (gnunicorn) [inline presentations/coaching/index.html:122]: Ha. It works here - the problem of variable screen-sizes ... 

Generally imho we should make the progress bar less intrusive but your remark is fair enough. Let's go with this for now.
